### ISSUE TITLE
Cross TLM communication 

### ISSUE THREAD

**Issue by psumesh** [tpoikela/uvm-python#40]
@tpoikela 
Can we use this python uvm package to create monitor and from analysis port receive transactions from sv uvm(cross packet transaction between two languages with tlm)?

**Comment by tpoikela** [comment]
No, unfortunately this is not possible directly with uvm-python. You need to write a module with signals then drive those signals from SV side with transaction info, then you can sample the signals on Python side. If you only have one interface, this should not be too much effort.
